sympy/functions/elementary/exponential.py

```diff
@@ -753,8 +753,8 @@ def taylor_term(n, x, *previous_terms):  # of log(1+x)
                 return powsimp((-n) * p * x / (n + 1), deep=True, combine='exp')
         return (1 - 2*(n % 2)) * x**(n + 1)/(n + 1)
 
-    def _eval_expand_log(self, deep=True, **hints):
-        from sympy import unpolarify, expand_log
+    def _eval_expand_log(self, deep=True, factor=False, **hints):
```

Comment:
I suspect this should be passed via `hints`

Reply:
please check the changes.